### PR TITLE
add description of environment variables to README and local.env.dist

### PR DESCRIPTION
**Pull request by briskt**
### Added
- Add descriptions of environment variables to README and local.env.dist

### Removed
- Removed variable `LOGGING_HANDLER`. We always set it to `stderr` and it isn't of much use any other way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,29 +269,38 @@ but you are also able to put them in the `'authproc.idp'` array in your
 
 Example (in `metadata/saml20-idp-hosted.php`):
 
+```php
+use Sil\PhpEnv\Env;
```
Comment (mtompset) [inline README.md:273]: Why is the \E in red?

File: README.md [hunk 2]
```diff
@@ -383,54 +393,67 @@ them to the `'authproc'` array in your `metadata/saml20-idp-hosted.php` file.
 
 Example (for `metadata/saml20-idp-hosted.php`):
 
-    use Sil\PhpEnv\Env;
-    use Sil\Psr3Adapters\Psr3SamlLogger;
+```php
+use Sil\PhpEnv\Env;
```
Comment (mtompset) [inline README.md:397]: Again with the red \E. Something is weird.

File: README.md [hunk 3]
```diff
@@ -383,54 +393,67 @@ them to the `'authproc'` array in your `metadata/saml20-idp-hosted.php` file.
...
+`idBrokerBaseUri` is the base URL for the id-broker API. It must be a full http URL.
+
+`idBrokerTrustedIpRanges` is a comma-separated list of CIDR-formatted IPv4 or IPv6 networks. The PHP
+client for the id-broker API performs a DNS check against this list. If the resulting IP address is
+not in any of the trusted ranges, it will not proceed with connection to the API.
 
 The `idpDomainName` parameter is used to assemble the Relying Party Origin
 (RP Origin) for WebAuthn MFA options.
 
+The `mfaSetupUrl` parameter is for the URL of where to send the user if they
```
Comment (mtompset) [inline README.md:438]: Nice reordering based on the sample code above.

File: README.md [hunk 4]
```diff
@@ -459,44 +482,83 @@ them to the `'authproc'` array in your `metadata/saml20-idp-hosted.php` file.
 
 Example (for `metadata/saml20-idp-hosted.php`):
 
-    use Sil\PhpEnv\Env;
-    use Sil\Psr3Adapters\Psr3SamlLogger;
+```php
+use Sil\PhpEnv\Env;
```
Comment (mtompset) [inline README.md:486]: Perhaps it is github thinking it is supposed to be an ANSI escape code, but highlighting it, because it isn't?

File: README.md [hunk 5]
```diff
@@ -459,44 +482,83 @@ them to the `'authproc'` array in your `metadata/saml20-idp-hosted.php` file.
...
+use Sil\Psr3Adapters\Psr3SamlLogger;
 
+$metadata['idp.example.com'] = [
     // ...
-
     'authproc' => [
-        10 => [
+        30 => [
```
Comment (mtompset) [inline README.md:492]: Why the change to 30?

File: README.md [hunk 6]
```diff
@@ -569,26 +631,64 @@ To check the status of the website, you can access this URL:
 
 ### SilDisco module for SAML Discovery
 
+A SimpleSAMLphp module containing a custom IdP Discovery class and four Authentication Processing
+filters. It is meant to be used as a SAML Hub, also known as a SAML Proxy. For more information, see
+the [Module Overview](./docs/overview.md) in the docs/ folder.
+
 #### Configuration
 
-Ensure the DYNAMO_* environment variables are set as shown in the local.env.dist file.
```
Comment (mtompset) [inline README.md:574]: So, DYNAMO variables aren't used any more?

File: local.env.dist [hunk 7]
```diff
@@ -1,91 +1,177 @@
...
-LOGGING_HANDLER=
+# SECRET_SALT should be set to a random string. It is used by SimpleSAMLphp when hashing values
+# such as a generated NameID attribute. Search for 'secretsalt' in SimpleSAMLphp documentation
+# for more information.
+SECRET_SALT=
 
-# Session store configuration. SESSION_STORE_TYPE can be "sql" or "phpsession"
-SESSION_STORE_TYPE=
-SESSION_DURATION=
+# === MySQL database config ===
```
Comment (mtompset) [inline local.env.dist:14]: Changing order of variables makes brain hurt more reading.

File: README.md (hunk 1 above)
Comment (jason-jackson) [reply README.md:273]: Probably github auto format

**Review by jason-jackson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (briskt) [reply README.md:273]: Your guess is as good as mine.

File: README.md (hunk 5 above)
Comment (briskt) [reply README.md:492]: It's the sequence number we typically use in production.

File: README.md (hunk 6 above)
Comment (briskt) [reply README.md:574]: They are, but they're described in the other doc. Wanted to keep this brief here.
